test/test_ops.py

```diff
@@ -456,6 +457,19 @@ def get_sample():
                         check_alias_annotation(name, (get_sample(),) + sample.args, sample.kwargs,
                                                func_type=func_type, aten_name=op.aten_name)
 
+                        # TODO: use script graph as well
```

Comment:
The dtype check above is incorrect. First, this test only runs in float32 and complexfloat32. See the definition of `_variant_ops` here: 

https://github.com/pytorch/pytorch/blob/0ceea7faf4f864ff834617150f1d58d9c1c56b2b/test/test_ops.py#L267

Second, the comment says it would only check if the dtype is float32 or int64 but the check is actually for float32 and int32.

Reply:
Okay, updated to say, only running in float32..